### PR TITLE
Docker container for s3front letsencrypt

### PR DESCRIPTION
**Pull request by jifeon**
I won't create PR to origin repo, because we have to refactor some things before, I will create issue and we'll do it when we have time

for review @AGresvig @apsavin

### CODE REVIEW

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+i=0
+
+# add slave databases for replication targets, REPLICATION_SLAVES are db links separated with comma
```
Comment (AGresvig) [inline docker/docker-entrypoint.sh]: ?

**Comment by AGresvig** [comment]
Fantastic shit!

**Comment by jifeon** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;MKHrNYVU&#x2F;378-update-s3-ssl-certificates"> Update S3 ssl certificates</a>

File: docker/docker-entrypoint.sh (hunk 1 above)
Comment (jifeon) [inline docker/docker-entrypoint.sh]: ahahah lol, looks like it's from my clipboard )))

**Pull request merged**